recipe/session/utils.go

```diff
@@ -249,28 +249,34 @@ func getCurrTimeInMS() uint64 {
 }
 
 func attachCreateOrRefreshSessionResponseToRes(config sessmodels.TypeNormalisedInput, res http.ResponseWriter, response sessmodels.CreateOrRefreshAPIResponse, tokenTransferMethod sessmodels.TokenTransferMethod) {
-	accessToken := response.AccessToken
 	refreshToken := response.RefreshToken
-	setFrontTokenInHeaders(res, response.Session.UserID, response.AccessToken.Expiry, response.Session.UserDataInAccessToken)
-	setToken(
-		config,
-		res,
-		sessmodels.AccessToken,
-		accessToken.Token,
-		// We set the expiration to 100 years, because we can't really access the expiration of the refresh token everywhere we are setting it.
-		// This should be safe to do, since this is only the validity of the cookie (set here or on the frontend) but we check the expiration of the JWT anyway.
-		// Even if the token is expired the presence of the token indicates that the user could have a valid refresh
-		// Setting them to infinity would require special case handling on the frontend and just adding 10 years seems enough.
```

Comment:
This comment is missing from the implementation below. Also, I think a constant would be nice for the magic number blow. (I'll change this in node as well)